src/Backend/Modules/MediaLibrary/Layout/Templates/MediaBrowser.html.twig

```diff
@@ -72,9 +72,9 @@
                           </form>
                           {% if not dataGrid.numberOfResults %}
                             {% if mediaFolder %}
-                              <p>{{ 'msg.NoItems'|trans|format(geturl('add', null, '&folder=#{mediaFolder.id}')) }}</p>
+                              {{ macro.emptyState(null, null, 'msg.NoItems'|trans|ucfirst) }}
```

Comment:
I think you can add the link here
`geturl('add', null, '&folder=#{mediaFolder.id}')`

Reply:
No thats wrong, add is not an action in de Media Library module. I also replaced it in another file with `MediaItemUpload` in `MediaLibrary/Layout/Templates/MediaItemIndex.html.twig`. The thing is, `MediaItemUpload` won't work here. Because you are in de browser. For the moment I will leave it like this. Need some customisation in de module itself to add a link to upload